### ISSUE TITLE
Global search

### ISSUE THREAD

**Issue by mbrpdr** [FreshRSS/FreshRSS#2131]
When a search is performed from the freshrss mainstream, it seams the search is limited to only mainstream enabled feeds. Is it possible to perform a global search across all feeds including those grouped in a separate category and not displayed in the mainstream?

**Comment by Alkarex** [comment]
Yes indeed, we need to find a good way to do that. There are a few options, such as:
1. Always search in all feeds
2. New search parameter, e.g. `minpriority:0`

**Comment by mbrpdr** [comment]
No, sorry

Always search all feeds could give problems I think when also using feeds from forums for example.

**Comment by squromiv** [comment]
> * Always search in all feeds
> * New search parameter, e.g. `minpriority:0`

May be a checkbox?
![screen_2019 03 23](https://user-images.githubusercontent.com/15985877/54858494-adf5ee00-4d0d-11e9-9720-ef18e56ed08a.png)

Or like this
![screen_2019 03 23 00111](https://user-images.githubusercontent.com/15985877/54859140-55751f80-4d12-11e9-9113-c516b11048a9.png)

Or in this way
![screen_2019 03 23 002221](https://user-images.githubusercontent.com/15985877/54859269-f0bac480-4d13-11e9-8402-0cd5945044b5.png)

![screen_2019 03 23 001777](https://user-images.githubusercontent.com/15985877/54859454-0af5a200-4d16-11e9-999c-0cb55ad5e6c9.png)

**Comment by aledeg** [comment]
Duplicate of #690

**Comment by crone123** [comment]
Hello,
Is this feature available (in this case, how to use it?), or planned to be added?
I am very interested about this...
Thank you :)

**Comment by Alkarex** [comment]
Implemented in https://github.com/FreshRSS/FreshRSS/pull/7144
